ArcGISRuntimeSDKQt_CppSamples/Maps/ApplyScheduledMapUpdates/ApplyScheduledMapUpdates.cpp

```diff
@@ -0,0 +1,216 @@
+// [WriteFile Name=ApplyScheduledMapUpdates, Category=Maps]
+// [Legal]
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#include "ApplyScheduledMapUpdates.h"
```

Comment:
Is this the name we settled on in the common design? `Scheduled Map Updates` doesn't sound correct to me. The updates are not to the map itself - but to the features. Can't it just be:

`ApplyPreplannedScheduledUpdates` or `ApplyScheduledUpdates` or `ApplyOfflineScheduledUpdates`?

Reply:
As discussed, the class/file names are just shortened names. I can change them if you think it's important, but the official name `Apply scheduled updates to preplanned map area` is what will show up in the viewer, on the website, and in the readme